landoapi/api/revisions.py

```diff
@@ -53,6 +62,29 @@ def _format_revision(
     Returns:
         A hash of the formatted revision information.
     """
+    # Load the revision's diff if requested.
+    diff = None
+    if include_diff and revision['activeDiffPHID']:
+        raw_diff = phab.get_diff(phid=revision['activeDiffPHID'])
+        diff = {
+            'id': int(raw_diff['id']),
+            'revision_id': int(raw_diff['revisionID']),
+            'date_created': int(raw_diff['dateCreated']),
+            'date_modified': int(raw_diff['dateModified']),
+            'vcs_base_revision': raw_diff['sourceControlBaseRevision'],
+            'authors': None
+        }
+
+        if raw_diff['properties'] and raw_diff['properties']['local:commits']:
```

Comment:
These two dict access attempts appear fragile: they will blow up if the phabricator API's JSON schema changes, or if the 'local:commits' property is not guaranteed to be present in all responses.  Is that by design?  If that's not the intention, it might be better to use `dict.get()`.